connectivity/suite.go

```diff
@@ -295,6 +295,7 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 
 	// Run a simple test with k8s Network Policy.
 	ct.NewTest("client-ingress-knp").WithK8SPolicy(clientIngressFromClient2PolicyKNPYAML).
+		WithFeatureRequirements(check.RequireFeatureDisabled(check.FeatureEndpointRoutes)).
```

Comment:
Could we also check for IPv6 too, and only disable if IPv6 is disabled? It seems with IPv4 only, the tests seem to pass, and I'd like to keep testing for KNP support e.g. on EKS.